openedx/core/djangoapps/zendesk_proxy/v0/views.py

```diff
@@ -39,21 +39,31 @@ def post(self, request):
         """
         request body is expected to look like this:
         {
-            "name": "John Q. Student",
-            "email": {
-                "from": "JohnQStudent@realemailhost.com",
-                "message": "I, John Q. Student, am having problems for the following reasons: ...",
-                "subject": "Help Request"
+            "requester": {
+                "email": "john@example.com",
+                "name": "name"
             },
-            "tags": ["zendesk_help_request"]
+            "subject": "test subject",
+            "comment": {
+                "body": "message details",
+                "uploads": ['file_token'],
+            },
+            "custom_fields": [
+                {
+                    "id": '001',
+                    "value": 'demo-course'
+                }
+            ],
+            "tags": ["LMS"]
```

Comment:
If you change this view method out from under us, [our studio-frontend code](https://github.com/edx/studio-frontend/blob/dca02351143d6626c07896b80f358a56c2708f5c/src/data/api/client.js#L87-L107) will break. Please either modify this method in a backwards-compatible way, or add a separate view.